builtin/game/chatcommands.lua

```diff
@@ -530,11 +530,16 @@ core.register_chatcommand("giveme", {
 })
 
 core.register_chatcommand("spawnentity", {
-	params = "<EntityName>",
-	description = "Spawn entity at your position",
+	params = "<EntityName> [<X>,<Y>,<Z>]",
```

Comment:
Shouldn't it be `[X, Y, Z]` instead? I thought <> means required and [] means optional.